walsh_ao5A.js

```diff
@@ -0,0 +1,43 @@
+/* Tyler Walsh
+walsh_ao5A.js
+19WI_INFO_2124_WW
+Thoendel
+1/18/2019 */
+
+const toDoList = []; //An array to keep track of my to-do-list. Created with an array literal.
+
+toDoList[0] = "Buy Groceries"; //The first item on my to-do-list.
+toDoList[1] = "Change Tire"; //The second item on my to-do-list.
+toDoList[2] = "Vacuum"; //The third item on my to-do-list.
+toDoList[3] = "Make Dinner"; //The fourth item on my to-do-list.
+// console.log(toDoList); Outputs the list to check what order things are in. Exactly as listed above.
```

Comment:
I'm not sure why you commented this out. These should output as part of the assignment.